public/index.html

```diff
@@ -44,18 +44,18 @@
 
             <div class="col-sm-12 col-md-12 col-lg-6 mb-3">
                 <div class="card border border-brown shadow">
-                    <div class="card-body text-center">
+                    <a class="card-body text-center" href="https://bscscan.com/token/0x0de08c1abe5fb86dd7fd2ac90400ace305138d5b" style="color: #4f4f4f;">
```

Comment:
Please use _global_variables.BSC_EXPLORER_ and _global_variables.BSC_CONTRACT_ here